src/backend/feed/worker.js

```diff
@@ -5,16 +5,19 @@ const { logger } = require('../utils/logger');
 
 const Post = require('../post');
 
```

Comment:
Remove this blank line.

Reply:
Removed the blank line

Thank you, 
Pavan